torch/quantization/fx/_equalize.py

```diff
@@ -85,15 +85,17 @@ def get_input_minmax(self):
     def set_equalization_scale(self, equalization_scale):
         # Reshape the equalization scale along axis=1 so that it can be
         # multiplied with the input along axis=1
+        if equalization_scale.nelement() == 1 and equalization_scale == torch.tensor(1):
+            return
         self.equalization_scale = torch.reshape(equalization_scale, self.equalization_shape)
 
     def calculate_scaled_minmax(self):
         r""" Returns the scaled min/max inputs
         """
-        if self.equalization_scale.nelement() == 0:
+        if self.equalization_scale.nelement() == 1 and self.equalization_scale == torch.tensor(1):
```

Comment:
Do we need to check for `self.equalization_scale.nelement() == 1`?

Reply:
I think we do, or else it will throw an error when comparing something of a different shape ex. a tensor array vs. the tensor(1).